osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -2717,6 +2717,36 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/crescent/images/cre.svg"
       },
       "coingecko_id": "crescent-network"
+    },
+    {
+      "description": "The native token of Oraichain",
+      "denom_units": [
+        {
+          "denom": "ibc/161D7D62BAB3B9C39003334F1671208F43C06B643CC9EDBBE82B64793C857F1D",
+          "exponent": 0,
+          "aliases": [
+            "orai"
```

Comment:
```suggestion
            "uorai"
```

Reply:
So if I'm not mistaken this is simply meta data redundancy being shown as what the relative exponents would be for a case of classing as a 10^-6 being of uorai exponent 0, and, orai exponent 6? Both being correct with the same hash, just nominal in difference? Just seeing if I understand as I haven't usually used both, if it is able to be.

Reply:
Yeah, this is purely aesthetic.  The alias can even be removed if you'd like

Reply:
i see, then i'll remove the alias to avoid confusion then